pkg/csi/service/wcp/controller.go

```diff
@@ -171,6 +174,9 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 		return err
 	}
 	go cnsvolume.ClearTaskInfoObjects()
+	if tasksListViewEnabled {
+		go cnsvolume.ClearInvalidTasksFromListView(false)
```

Comment:
Inside this method `ClearInvalidTasksFromListView` there is a dependency on another FSS. Have we evaluated the feature inter-op when one of them is disabled/enabled ? If yes, can you please share some details in this thread

Reply:
Multi-vc feature is only present in Vanilla at the moment. For WCP, we only have single VC. 
So, the multi-vc FSS can't be be passed to this method like it has been in the vanilla controller init code -> https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/csi/service/vanilla/controller.go#L310. 

From the https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/1949 description, 
> This fss will also work alongside the multi-vcenter fss change such that either of them can be set independently.